syntax = "proto3";

package chatroom.msgpb;

message GetFriends {
    int32 user_id = 1;
}